src/components/Card/Card.test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable testing-library/no-container */
+/* eslint-disable testing-library/no-node-access */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Card } from './Card';
+
+describe('Test Card', () => {
+  it('should render Card component without error', () => {
+    render(<Card />);

```

Comment:
Add `expect`. You can use toMatchSnapshot in case you just want to test your component renders successfully. 
https://jestjs.io/uk/docs/snapshot-testing